dyn_sim/ctrl/segway/fdbk_lin_segway_ctrl.py

```diff
@@ -0,0 +1,109 @@
+import numpy as np
+
+from dyn_sim.ctrl.ctrl import Controller
+from dyn_sim.sys.planar.segway import Segway
+
+# constants
+g = 9.80665  # gravitational acceleration
+
+
+class FLPosSegwayController(Controller):
+    """Feedback linearizing controller for segway.
+
+    Only capable of position regulation at the moment.
+    Could implement trajectory following using Callables.
+
+    Derivation of feedback linearizing controller available in
+    'CDS233: Project Submission 2'
```

Comment:
this isn't super helpful since the file isn't going to be readily available. either bite the bullet and explain the steps in the comments, or just hope that whoever is reading this knows what lie derivatives are.